eZ/Publish/SPI/Search/Capable.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * This file is part of the eZ Publish Kernel package.
+ *
+ * @copyright Copyright (C) eZ Systems AS. All rights reserved.
+ * @license For full copyright and license information view LICENSE file distributed with this source code.
+ */
+namespace eZ\Publish\SPI\Search;
+
+/**
+ * Capability interface for search engines needed for {@see eZ\Publish\API\Repositor\SearchService::supports()}.
+ *
+ * @since 6.12 And ported to 6.7.6 for search engine forward compatibility.
```

Comment:
note to self, the whole Repository change should not go into 6.7 as it adds repository method, not something we do in patches.